tests/precompile/contracttest/example_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Comment:
This file is an example of how testing can be done directly on libevm's `simulated.Backend,` without doing a tmpnet, RPC, etc. 

It's cleaner and faster to execute. It's also currently unworkable.